resources/language/German/strings.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
+  <!-- Main -->
+  <string id="1000">Hauptmenü</string>
+
+  <string id="1100">Bridge Konfiguration</string>
+  <string id="1101">Starte automatische Kofiguration</string>
```

Comment:
was that it? i just changed the file online within github

Martin Polednik <notifications@github.com> schrieb am Do., 7. Juli 2016 um
12:29 Uhr:

> In resources/language/German/strings.xml
> <https://github.com/mpolednik/script.kodi.hue.ambilight/pull/11#discussion_r69884483>
> :
>
> > @@ -0,0 +1,98 @@
> > +<?xml version="1.0" encoding="utf-8" standalone="yes"?>
> > +<strings>
> > +  <!-- Main -->
> > +  <string id="1000">Hauptmenü</string>
> > +
> > +  <string id="1100">Bridge Konfiguration</string>
> > +  <string id="1101">Starte automatische Kofiguration</string>
>
> Please update the PR with fixed commit (if it's wrong, my german sucks).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mpolednik/script.kodi.hue.ambilight/pull/11/files/8c3cf37a90a5ec77a665a53bd3a7560226864195#r69884483>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABa1EqwfmdjVe45CRD0j8EnyWgqyH-OGks5qTNTygaJpZM4JEHoc>
> .
>